test/suite/test_export02.py

```diff
@@ -0,0 +1,110 @@
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+#
+
+from helper import copy_wiredtiger_home
+from helper_tiered import TieredConfigMixin, tiered_storage_sources
+from wtscenario import make_scenarios
+import os, wttest
+
+class test_export02(TieredConfigMixin, wttest.WiredTigerTestCase):
+    dir = 'backup.dir'
+
+    types = [
+        ('table', dict(type = 'table:')),
+    ]
+
+    scenarios = make_scenarios(tiered_storage_sources, types)
+
+    # Test that the WiredTiger.export file contains the correct contents
+    # after performing operations on the home directory, creating a backup
+    # directory, and then performing operations on the backup directory.
+    def test_export(self):
+        uri_d = self.type + "exportd"
+        uri_e = self.type + "exporte"
+        uri_f = self.type + "exportf"
+
+        # Create two tables.
+        self.session.create(uri_d)
+        self.session.create(uri_e)
+
+        # Insert some records.
+        c4 = self.session.open_cursor(uri_d)
+        c4["k4"] = "v4"
+        c4.close()
+
+        c5 = self.session.open_cursor(uri_e)
+        c5["k5"] = "v5"
+        c5.close()
+
+        self.session.checkpoint()
+
+        if self.is_tiered_scenario():
+            self.session.flush_tier(None)
+
+        # Open a special backup cursor for export operation.
+        main_cursor = self.session.open_cursor('backup:export', None, None)
+
+        # Copy the main database to another directory, including the WiredTiger.export file.
+        os.mkdir(self.dir)
+        copy_wiredtiger_home(self, '.', self.dir)
+
+        main_cursor.close()
+        self.close_conn()
+
+        # Open a connection and session on the directory copy.
+        self.conn = self.setUpConnectionOpen(self.dir)
+        self.session = self.setUpSessionOpen(self.conn)
+
+        # Create a third table and drop the second table.
+        self.session.create(uri_f)
+        c6 = self.session.open_cursor(uri_f)
+        c6["k6"] = "k6"
+        c6.close()
+
+        self.session.checkpoint()
+
+        if self.is_tiered_scenario():
+            self.session.flush_tier(None)
+
+        self.session.drop(uri_e)
+
+        # Open an export cursor on the database copy.
+        export_cursor = self.session.open_cursor('backup:export', None, None)
+        self.assertTrue(os.path.isfile(self.dir + "/WiredTiger.export"))
+
+        # The information for the third table should exist in the WiredTiger.export file
+        # but the information for the second table should not exist in the file.
+        self.assertTrue(open(self.dir + "/WiredTiger.export", "r").read().find("exporte") == -1)
```

Comment:
Is there any way to turn one of these into an `assertFalse` instead? Trying to notice that 1 character difference in the two lines is visually tricky.

Reply:
Fair point, I'll change the first statement to use `assertFalse` instead.